simple_history/tests/tests/test_models.py

```diff
@@ -1441,6 +1445,9 @@ def test_foreign_key_to_self_using_self_str(self):
 
 @override_settings(**database_router_override_settings)
 class MultiDBExplicitHistoryUserIDTest(TestCase):
+    if django.VERSION >= (2, 2):
```

Comment:
Why is this version spec different from the other two in this file? And if it's not supposed to be, could you refactor it out into a function?